magento/pwa-studio
# [doc] update getting started section

## Description

This PR makes the following changes to the doc project:

- Renames the _Getting Started_ section to _Overview_ to reduce confusion
- Reorganizes the navigation in that section
- Update the _Overview_ topic to better reflect the current state of the project and link to other topics
- Remove deprecated navigation files

## Related Issue

Closes PWA-1191

## Acceptance

@davemacaulay 

### Verification Stakeholders

@davemacaulay 

### Specification



### Verification Steps
1. Navigate to the `pwa-devdocs` directory
2. Run the HTML preview server: `yarn develop`
3. Click on the `Overview` card
4. Verify you are navigated to the Overview topic
5. Verify section navigation makes sense

## Screenshots / Screen Captures (if appropriate)

![Screen Shot 2021-01-05 at 10 09 40 AM](https://user-images.githubusercontent.com/4692281/103669621-31fa8380-4f3e-11eb-88c8-6f3c0b6a4494.png)

![Screen Shot 2021-01-05 at 11 08 43 AM](https://user-images.githubusercontent.com/4692281/103676198-6b36f180-4f46-11eb-852b-310044efc9c2.png)


## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
